### PR TITLE
Add a "Report a problem" button to the footer

### PR DESCRIPTION
**Pull request by GuelorEmanuel**
#138

### CODE REVIEW

File: src/templates/search.jsx
Comment (halkeye) [inline src/templates/search.jsx]: So this will make a path polyfill be added to the front-end bundle. And I'm really not sure what the path is doing. Is it just for issue template? I don't think it needs to be anything but a simple string.

**Review by halkeye — COMMENTED** [review]
@oleg-nenashev should weigh as the original requester, but I don't think it needs to be added to every page, then you can just report `window.location.href` and not worry about all the other items that will make `<Layout` more complicated than it is. and `document.querySelector('title').textContent` to get the title

But that would make it a runtime thing, instead of part of the static html... so maybe not.

File: utils.js
Comment (halkeye) [inline utils.js]: Can you make a new component, i'm sad how big and complicated I had to make this function.

see the `$('#creativecommons').append('<SiteVersion />');` above
Something like `$('#.license-box').append('<ReportAProblem />');` (or even after().append() type thing)

File: src/templates/search.jsx
Comment (GuelorEmanuel) [reply src/templates/search.jsx]: That's true, I am not too sure what I was thinking when I added this. Having it as a simple string does make sense. Thanks!

File: utils.js
Comment (GuelorEmanuel) [reply utils.js]: Interesting, great suggestion. I will refactor this by creating a new component. Thanks @halkeye

File: utils.js
Comment (halkeye) [inline utils.js:119]: Not a big deal here, and I'll update eslintrc to prevent end of line space, but in general watch out for PRs with unrelated changed lines.

File: src/components/ReportAProblem.jsx
Comment (halkeye) [inline src/components/ReportAProblem.jsx]: i don't think the title attribute is right here, i'll leave it up to you though

https://html.com/attributes/a-title/

I'm pretty sure when you hover over this link it'll say something like "Plugin page not found" which I don't think is right for the tooltip, I think the default of "Report a problem" makes more sense.

**Review by halkeye — APPROVED** [review]
Thank you for this.

Overall I'm pretty happy with it. I'm going to think about if there's a gatsby way to do the <layout stuff without needing the 4 parameters everywhere, but at the moment I can't think of it.

I'll wait to see if anyone else has comments, but LGTM

**Review by timja — APPROVED** [review]
Nice, my only worry is that this will get reports of issues with plugin docs, rather than the plugin site itself,

It’s worth trialing anyway to see what we get, possibly routing issues to maintainers is good through here, and easy to turn off if we want

File: src/components/ReportAProblem.jsx
Comment (GuelorEmanuel) [reply src/components/ReportAProblem.jsx]: I'll update this in a bit.

**Comment by halkeye** [comment]
WFM merging

**Review by oleg-nenashev — COMMENTED** [review]
Thanks for the patch!

**Pull request merged**
